app/scripts/services/validateRegistrant.js

```diff
@@ -8,16 +8,19 @@ angular.module('confRegistrationWebApp')
       var answers = registrant.answers;
       angular.forEach(block.rules, function(rule){
         var answer = _.find(answers, {blockId: rule.parentBlockId});
-        if(angular.isUndefined(answer) || answer.value === ''){
+        if(angular.isUndefined(answer) || answer === null || answer.value === ''){
           returnValue = false;
         }else{
-          if(rule.operator === '=' && answer.value !== rule.value) {
+          //If string is a number, parse it as a float for numerical comparison
+          var answerValue = !isNaN(answer.value) ? parseFloat(answer.value) : answer.value;
+          var ruleValue = !isNaN(rule.value) ? parseFloat(rule.value) : rule.value;
```

Comment:
Ya, you're probably right. Oh well. I could split it out if this doesn't get merged immediately.